acceptance-test/src/acceptance-test/java/org/zalando/nakadi/webservice/hila/HilaAT.java

```diff
@@ -751,22 +817,37 @@ public void whenIsLookingForDeadLetterAndSendAllEventsOneByOneThenBackToNormalBa
     }
 
     @Test(timeout = 20_000)
-    public void shouldSkipDeadLetterdAndConsumptionToBeContinued() throws IOException {
+    public void shouldSkipDeadLetterAndConsumptionToBeContinuedAndDeadLetterFoundInTheQueueLater()
+            throws IOException, InterruptedException {
+
         final EventType eventType = NakadiTestUtils.createBusinessEventTypeWithPartitions(4);
 
         publishBusinessEventWithUserDefinedPartition(eventType.getName(),
-                50, i -> String.format("{\"foo\":\"bar%d\"}", i), i -> String.valueOf(i % 4));
+                50, i -> String.format("bar%d", i), i -> String.valueOf(i % 4));
+
+        final String poisonPillValue = "bar10";
```

Comment:
trying to understand why `bar10` specifically is the poison pill and not the values before it?

Reply:
> trying to understand why `bar10` specifically is the poison pill and not the values before it?

We just want some value in the middle of the stream.  We may even select it randomly, but that'd make it harder to debug if the test stops working for some reason.

Reply:
I took a look again and now understand the logic.